pkg/traces/exporter/client_authenticator.go

```diff
@@ -22,6 +24,12 @@ func (c *clientAuthenticator) setToken(token string) {
 	c.token = token
 }
 
+// setDisableTLS updates the return value of RequireTransportSecurity. TLS should
+// be disabled only for local testing.
+func (c *clientAuthenticator) setDisableTLS(disableTLS bool) {
+	c.disableTLS = disableTLS
+}
```

Comment:
If we only need this for tests, we can define this in the `_test.go` file.

Reply:
I actually don't want it only for tests! `RequireTransportSecurity` is part of the `google.golang.org/grpc/credentials.PerRPCCredentials` interface and I'd been just hardcoding the return to false for initial testing. But now that we've got a proper flag for `disable_ingest_tls` I wanted to plumb that through here as well.

Reply:
(By "local testing" in the comment I mean that we would only set `disable_ingest_tls` to false when testing against e.g. a local collector running in docker.)

Reply:
okay